### PR TITLE
Docs prep style guide

### PR DESCRIPTION
**Pull request by stevepiercy**
This is work to resolve #3327 and #2880.

I would like review of the issue templates. I don't know if we want all of them, or merge them into a single template, or any other revision. You can test drive them on [docs-style-guide](https://github.com/Pylons/docs-style-guide/issues/new/choose).

### CODE REVIEW

**Comment by stevepiercy** [comment]
I made a guess in `setup.cfg`, due to a [failed build](https://travis-ci.org/Pylons/pyramid/jobs/431769037). Please let me know if that is not correct.

File: setup.cfg [hunk 1]
```diff
@@ -69,6 +69,8 @@ show-source = True
 
 [check-manifest]
 ignore =
+    .github
+    .github/*
```
Comment (mmerickel) [inline setup.cfg]: these should be added to the manifest, not excluded

File: MANIFEST.in [hunk 2]
```diff
@@ -13,4 +13,6 @@ include HACKING.txt hacking-tox.ini
 include builddocs.sh coverage.sh scaffoldtests.sh
 include TODO.txt
 
+include .github .github/*
```
Comment (mmerickel) [inline MANIFEST.in]: graft

Comment (stevepiercy) [reply MANIFEST.in]: I don't understand "graft". Sorry.

I read https://pypi.org/project/check-manifest/ and got this:

```
missing from sdist:
  .github
  .github/ISSUE_TEMPLATE
  .github/ISSUE_TEMPLATE/bug_report.md
  .github/ISSUE_TEMPLATE/documentation.md
  .github/ISSUE_TEMPLATE/feature_request.md
don't know how to come up with rules matching any of the files, sorry!
```

So I tried that locally as well, it failed but for different reasons, but I committed and pushed. We'll see if this guess is right. I have no clue what I'm doing.

Comment (mmerickel) [reply MANIFEST.in]: https://docs.python.org/2/distutils/sourcedist.html#commands read the docs and run `tox -e lint` locally if you aren't sure before you push.

Comment (stevepiercy) [reply MANIFEST.in]: `tox -e lint` always fails for me locally with a bajillion `*.DStore`s, so I don't pay it any mind:

```
missing from VCS:
lists of files in version control and sdist do not match!
missing from VCS:
  docs/.DS_Store
  docs/_templates
  docs/_templates/.DS_Store
...
```

Comment (stevepiercy) [reply MANIFEST.in]: OK, I think this is good now. Thanks for the link to the docs. Had to play around a bit to see what fails locally vs. in Travis.

**Pull request merged**